cli/docker.go

```diff
@@ -620,7 +642,9 @@ func runDockerCVM(ctx context.Context, log slog.Logger, client dockerutil.Client
 			mountpoint := bind.Path
 			if strings.HasPrefix(mountpoint, flags.hostUsrLibDir) {
 				mountpoint = filepath.Join(
-					"/usr/lib",
+					// Note: we used to mount into /usr/lib, but this can change
+					// based on the distro inside the container.
+					imgMeta.UsrLibDir(),
```

Comment:
I think we should still have an env var for this in case the image doesn't have an os-release or something. Most Coder templates only use a single image (or at the very least the same image family) so it would add an easy way of overriding the auto-detected value

Reply:
Added `CODER_INNER_USR_LIB_DIR`.